scavenger-model/src/main/java/com/navercorp/scavenger/util/HashGenerator.java

```diff
@@ -1,37 +1,58 @@
 package com.navercorp.scavenger.util;
 
+import org.apache.commons.codec.digest.MurmurHash3;
+
 import java.math.BigInteger;
 import java.nio.charset.StandardCharsets;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
 
 public class HashGenerator {
-    public static class Sha256 {
+
+    public static class DefaultHash {
         public static String from(String signature) {
+            return Murmur.from(signature);
```

Comment:
Changing the hash algorithm is a big task that requires migrating DB data. 
It's better to keep `md5` as it is.

Reply:
Good point, I forgot to take backwards compatibility into consideration

How would you feel if I made it a command line option instead (retaining MD5 as default)? I can extract this change into a separate PR

Reply:
Yes it seems like a good idea to default to md5 and add it as an option for the user to configure.

cc. @kojandy @sohyun-ku

Reply:
Extracted into #142